client/bootstrap

```diff
@@ -682,6 +684,86 @@ class Node(object):
         '''
         self.server_.get_resource(url, path)
 
+
+    def create_user(self, user, group, enc_passwd,
+                    root='/persist/local/', ssh_keys=None):
+        '''Create a local user on the bootstrapped node.
+        You can specify:
+         - user: the username
+         - group: the group assigned to the user
+         - enc_passwd: an encrypted password assigned to the user
+         - root: the path where the user's home directory will reside
+         - ssh_keys: public keys that will be copied to ~$HOME/.ssh/authorized_keys
+
+        The method will fail if the user already exists on the node. The
+        $HOME/.ssh directory will be assigned 0700 permissions and the
+        $HOME/.ssh/authorized_keys file will be assigned 0600 permissions in
+        accord with SSH best practices.
+
+        Raises:
+            subprocess.CalledProcessError:
+             -
+            OSError
+             -
+
+        Returns:
+            dict: User details
```

Comment:
why? If it fails, will raise exception, otherwise the caller passed the details as arguments, so he already knows them